src/test/java/net/mcreator/integration/generator/GTSampleElements.java

```diff
@@ -89,6 +89,20 @@ public static void provideAndGenerateSampleElements(Random random, Workspace wor
 				workspace.getModElementManager().storeModElement(procedure);
 			}
 
+			for (int i = 1; i <= 4; i++) {

```

Comment:
There are string procedures being prepared right below (string1 and string2) that are already used by eg. GUI tests. Re-ruse those elements

Reply:
Done!